packages/react-components/src/components/MessageThread.tsx

```diff
@@ -1177,6 +1178,10 @@ export const MessageThreadWrapper = (props: MessageThreadProps): JSX.Element =>
   // To rerender the defaultChatMessageRenderer if app running across days(every new day chat time stamp need to be regenerated)
   const defaultChatMessageRenderer = useCallback(
     (messageProps: MessageProps) => {
+      /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+      if (inlineAttachments[messageProps.message.messageId] === undefined) {
+        setInlineAttachments((prev) => ({ ...prev, [messageProps.message.messageId]: {} }));
+      }
```

Comment:
In conjunction in the comment above for line 920, this `if` check here can be removed? since if it's not in the dictionary/object, it will take `inlineAttachments[messageProps.message.messageId] ?? {}` to be empty automatically?